app/models/user.rb

```diff
@@ -34,12 +36,33 @@ def remember
     update_attribute :remember_digest, User.digest(remember_token)
   end
 
-  def authenticated? remember_token
-    return false unless remember_digest
-    BCrypt::Password.new(remember_digest).is_password?remember_token
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
+    return false if digest.nil?
+    BCrypt::Password.new(digest).is_password? token
   end
 
   def forget
     update_attribute :remember_digest, nil
   end
+
+  def activate
+    update_attributes :activated, true
+    update_attributes :activated_at, Time.zone.now
```

Comment:
```suggestion
    update_attributes activated_at: Time.zone.now, activated: true
```
remove update_attributes :activated, true, we can update two attributes in same time by update_attributes
you can learn about other types of updates like: update_attribute, update_column, update_columns

Reply:
Noted it, thank you.